CONTRIBUTING.md

```diff
@@ -82,74 +82,60 @@ web application.
 ```
-Connect is [DOWN]
-Control Center is [DOWN]
-Kafka is [DOWN]
-Kafka REST is [DOWN]
-ksqlDB Server is [DOWN]
-Schema Registry is [DOWN]
-ZooKeeper is [DOWN]
+    Name                    Command               State                               Ports
+--------------------------------------------------------------------------------------------------------------
+broker            /etc/confluent/docker/run        Up      0.0.0.0:9092->9092/tcp,:::9092->9092/tcp,
+0.0.0.0:9101->9101/tcp,:::9101->9101/tcp
+connect           /etc/confluent/docker/run        Up      0.0.0.0:8083->8083/tcp,:::8083->8083/tcp, 9092/tcp
+control-center    /etc/confluent/docker/run        Up      0.0.0.0:9021->9021/tcp,:::9021->9021/tcp
+ksql-datagen      bash -c echo Waiting for K ...   Up
+ksqldb-cli        /bin/sh                          Up
+ksqldb-server     /etc/confluent/docker/run        Up      0.0.0.0:8088->8088/tcp,:::8088->8088/tcp
+rest-proxy        /etc/confluent/docker/run        Up      0.0.0.0:8082->8082/tcp,:::8082->8082/tcp
+schema-registry   /etc/confluent/docker/run        Up      0.0.0.0:8081->8081/tcp,:::8081->8081/tcp
 ```
 
-The Kafka [Datagen Source Connector](https://www.confluent.io/hub/confluentinc/kafka-connect-datagen) is a convenient 
-tool for local development and testing. Install it via the following:
-
-    confluent-hub install confluentinc/kafka-connect-datagen:0.6.0
-
-Then build and install the connector to the Confluent Platform indicated by your $CONFLUENT_HOME environment variable:
-
-    ./gradlew clean installConnectorInConfluent
-
-Note that any time you modify the MarkLogic Kafka connector code, you'll need to repeat the 
-`./gradlew clean installConnectorInConfluent` step. Note that `clean` is included to ensure that in case you've changed 
-any connector dependencies, old dependencies will not be included in the connector archive.
-
-Next, start Confluent:
+You can now visit http://localhost:9021 to access [Confluent's Control Center](https://docs.confluent.io/platform/current/control-center/index.html) application.
 
-    confluent local services start
-
-To verify that your Confluent installation is running properly, you can run `confluent local services status` and
-see logging similar to this:
-
-```
-Connect is [UP]
-Control Center is [UP]
-Kafka is [UP]
-Kafka REST is [UP]
-ksqlDB Server is [UP]
-Schema Registry is [UP]
-ZooKeeper is [UP]
-```
+Within Control Center, click on "controlcenter.cluster" to access the configuration for the Kafka cluster.
 
-You can now visit http://localhost:9021 to access [Confluent's Control Center](https://docs.confluent.io/platform/current/control-center/index.html)
-application.
+### Build and install the MarkLogic Kafka Connector
+1. Build the connectorArchive target using ```./gradlew installConnectorInConfluent```
+2. Restart the "connect" server in the Docker "confluent-platform-example" cluster.
+3. Verify the connector has loaded properly
```

Comment:
A nit - put a period at the end of everything (the docs team told me to do that, so I nearly always do it now).